packages/ui/src/modal/bid/BidModalRenderer.tsx

```diff
@@ -116,7 +119,8 @@ export const BidModalRenderer: FC<Props> = ({
   const { data: wallet } = useWalletClient()
   const [bidStep, setBidStep] = useState<BidStep>(BidStep.SetPrice)
   const [transactionError, setTransactionError] = useState<Error | null>()
-  const [bidAmount, setBidAmount] = useState<string>('')
+  const [bidInput, setBidInput] = useState<string>('')
```

Comment:
Why is this a part of the renderer? Shouldn't it live in the UI?

Reply:
Updated the variable name to `bidAmountPerUnit` to make it more clear